dags/search_rollup.py

```diff
@@ -15,12 +15,11 @@
         'telemetry-alerts@mozilla.com',
         'amiyaguchi@mozilla.com',
         'harterrt@mozilla.com',
-        'spenrose@mozilla.com'
     ],
     'email_on_failure': True,
     'email_on_retry': True,
     'retries': 2,
-    'retry_delay': timedelta(minutes=30),
+    'retry_delay': timedelta(hours=2),
```

Comment:
Is this something that's relevant just for this DAG's task instance and not for the task in main summary?

Reply:
Actually, this doesn't do what I meant it to do. This should be omitted.